frontend/src/Components/Navbar/index.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import firebase from "firebase";
+import { Link } from "react-router-dom";
+import "./style.css";
+
+// Depending on the current path, this component sets the "active" class on the appropriate navigation link item
+function Navbar() {
+  return (
+    <nav className="navbar navbar-expand-lg navbar-light bg-light justify-content-between">
+      <Link className="navbar-brand" to="/user">
+      <img
+              alt="profile"
+              src={firebase.auth().currentUser.photoURL}
+            />
+      </Link>
+      <span>Welcome {firebase.auth().currentUser.displayName}</span>
+      <div>
```

Comment:
Could probably extract out this div block to its own component and name it NavBarItems or NavBarLinks.